src/Xamarin.Android.Build.Tasks/Tasks/JavaDependencyVerification.cs

```diff
@@ -212,14 +213,12 @@ public void AddIgnoredDependencies (ITaskItem []? tasks)
 	{
 		foreach (var task in tasks.OrEmpty ()) {
 			var id = task.ItemSpec;
-			var version = task.GetRequiredMetadata ("AndroidIgnoredJavaDependency", "Version", log);
 
-			if (version is null)
-				continue;
-
-			if (version != null && MavenExtensions.TryParseArtifactWithVersion (id, version, log, out var art)) {
-				log.LogMessage ("Ignoring Java dependency '{0}:{1}' version '{2}'", art.GroupId, art.Id, art.Version);
-				artifacts.Add (art.ArtifactString, art);
+			if (MavenExtensions.TryParseArtifacts (id, log, out var parsed)) {
+				foreach (var art in parsed) {
+					log.LogMessage ("Ignoring Java dependency '{0}:{1}' version '{2}'", art.GroupId, art.Id, art.Version);
```

Comment:
Given that we're now standardizing on "group:id:version" within `%(JavaArtifacts)`, should we use that here as well?